src/samples/UnAuthChildBenefitsClaim/index.tsx

```diff
@@ -61,11 +61,18 @@ function staySignedIn(setShowTimeoutModal, refreshSignin = true) {
   initTimeout(setShowTimeoutModal);
 }
 
+function fetchClaimsData() {
+  PCore.getDataPageUtils().getDataAsync('D_ClaimantWorkAssignmentChBCases', 'root').then(
```

Comment:
'D_ClaimantWorkAssignmentChBCases'
Should be:  
D_GetUnauthClaimStatusBySessionID